src/ui-test/resources/JBang.yaml

```diff
@@ -0,0 +1,17 @@
+# camel-k: language=yaml
+
+# Write your routes here, for example:
+- from:
```

Comment:
what about to provide more specific file name than `JBang.yaml` to usecase we are testing?